content/learning-with-4geeks.md

```diff
@@ -0,0 +1,76 @@
+# Learning with 4Geeks
+
+The world is constantly evolving, and it's high time to shake things up and leave outdated teaching methods behind. We're here to unlock your full potential and have fun while doing it!
+
+At 4Geeks, we're all about making learning a fun and fulfilling experience! We believe that education should be hands-on, motivating, and tailored for people just like you. That's why we've come up with our own teaching method that combines all the best. Our ***[special formula](https://4geeksacademy.notion.site/4geeksacademy/Mastering-Technical-Knowledge-984d2df394c44aedb05987311ccfcf06)*** focuses on **Memory Retention** 🧠, **Feedback Quality** 🗣️, **Learning Mastery** 💡, and **Motivation** 💪 - all the ingredients for a well-rounded and effective learning journey. We've put in the hard work, collaborated with experts, and created an environment where you can truly thrive.
+
+## 🥇 Certifications and Courses
+
+Hey! We totally get it, life can get pretty busy sometimes, right? That's why we've designed two types of experiences: **Open Learning** and **premium Bootcamps**.
+Let's break it down and explore the benefits of each option, so you can choose the best fit for your goals and needs. Whether you prefer a flexible and self-paced learning journey or a more immersive and structured experience, we have something for you.
+
+### 1) Open Learning:
+
+This experience is perfect for those who do not have much time on their agenda but still want to have complete learning about a specific topic. These are small courses that last between 1 and 2 weeks. You will be able to join any of them for free in different technologies like Java, Python, Node, etc.
+
+We have listed some benefits that this type of experience has to offer:
+ 
+- **Flexibility**: Set your own pace and choose when and where you want to study.
+- **Variety of Courses**: Explore diverse topics and choose the courses that align with your interests and career goals.
+- **Self-Paced Learning**: Freedom to progress at your own speed, ensuring that you thoroughly grasp the concepts before moving on to the next topic.
+- **Workshops**: Access to live workshops from industry experts every week (included with subscription)
+- **1-1 Mentorship**: Personalized support that provides you guidance and feedback tailored to your needs.
```

Comment:
Add a better explaination of what 1-1 mentorships are, this is like the game changer of what we offer. 1-1 mentorships are meeting with experts from the tech industry that will help you understanding hard concepts, help you. to get unstuck and more. Check the landing: https://4geeks.com/mentorships